packages/amplify-app/src/xcodeHelpers.js

```diff
@@ -175,7 +175,7 @@ async function addAmplifyFiles() {
   }
   const rootDir = path.resolve(projectDir, '..', '..');
   const project = xcode.project(projectDir);
-  return new Promise((resolve, reject) => {
+  await new Promise((resolve, reject) => {
```

Comment:
I think it's preferable to let the caller decide whether to await. What's the reasoning behind this change?

Reply:
In either case the function returns `Promise<void>`. It was just a matter of deciding whether to have line 174 state `return undefined;` or explicitly defining a type on line 178. The promise on line 178 can only possibly resolve to `undefined` (see line 236).